packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -3632,6 +3632,83 @@ describe('plugin-meetings', () => {
         });
       });
 
+      describe(`#beRightBack`, () => {
```

Comment:
Hey @brycetham, I had a discussion with @antsukanova about the placement of these tests. The reason why these tests are placed here is because we are testing the public API. Previously, these tests were placed under the `[true, false].forEach((isMultistream) => describe('addMedia/updateMedia semi-integration tests ...`. However, these tests are specifically for adding and updating media, as well as publishing streams. This way, the tests are aligned with the other public APIs' tests.